create_box/main.py

```diff
@@ -1,9 +1,30 @@
 """This is the entry point of the program."""
 
+def create_box(height, width, char):
+    count = 0
+    end_str = ''
+    while count < height:
```

Comment:
The more pythonic way of doing this loop is to use something like `for h in range(height):` This will loop through exactly `height` times without needing to maintain a counter in your code.